### PR TITLE
 Delegate buffer writing generic Collection method 

### PR DESCRIPTION
**Pull request by aciidb0mb3r**

### CODE REVIEW

File: Sources/Basic/OutputByteStream.swift [hunk 1]
```diff
@@ -51,73 +51,82 @@ public class OutputByteStream: TextOutputStream {
...
     }
 
     /// Write an individual byte to the buffer.
-    public func write(_ byte: UInt8) {
+    public final func write(_ byte: UInt8) {
         buffer.append(byte)
     }
 
+    /// Write a collection of bytes to the buffer.
+    public final func write<C: Collection where C.Iterator.Element == UInt8>(collection bytes: C) {
```
Comment (ddunbar) [inline Sources/Basic/OutputByteStream.swift]: Was the parameter name required for some reason? I'd rather not have it for consistency.

Comment (aciidb0mb3r) [inline Sources/Basic/OutputByteStream.swift]: Yeah otherwise it becomes ambigous

Comment (ddunbar) [inline Sources/Basic/OutputByteStream.swift]: Is it possible to only have it for the sequence method?

Comment (aciidb0mb3r) [inline Sources/Basic/OutputByteStream.swift]: looks like it can be removed for sequence but not for collection

Comment (ddunbar) [inline Sources/Basic/OutputByteStream.swift]: Ok, I can live with it.

**Comment by ddunbar** [comment]
LGTM

**Pull request merged**